reverseCase/reverseCase.js

```diff
@@ -2,4 +2,18 @@
  * Write a function that reverses the case of each letter in the strings that it receives.
  * Example: 'Hello World' -> 'hELLO wORLD'
  * Assume that each string will contain only spaces and letters.
- */
\ No newline at end of file
+ */
+
+ const reverseCase = (str) => {
+   const words = str.split(' ');
+   const reverse = []
+   for (let i = 0; i < words.length; i++) {
+     for (let j = 0; j < words[i].length; j++) {
```

Comment:
Nice solution. I think you could find a way to only have to loop over the string once :) another caveat is that you don't need to turn the string into an array. You can loop over a string and simply treat it like an array :)